### PR TITLE
[WOR-646] Add enum and list cloudPlatform under required

### PR DESCRIPTION
**Pull request by ghost**
Ticket: https://broadworkbench.atlassian.net/browse/WOR-646
<Put notes here to help reviewer understand this PR>
- add enums in swagger for cloudPlatforms

---

**PR checklist**

- [ ] Include the JIRA issue number in the PR description and title
- [ ] Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] If you changed anything in `model/`, then you should [publish a new official `rawls-model`](https://github.com/broadinstitute/rawls/blob/develop/README.md#publish-rawls-model) and update `rawls-model` in [Orchestration's dependencies](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/project/Dependencies.scala).
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green, including CI tests
- [ ] **Squash commits and merge** to develop (branches are automatically deleted after merging)
- [ ] Inform other teams of any substantial changes via Slack and/or email

### CODE REVIEW

**Comment by ghost** [comment]
jenkins retest

**Comment by yunip20** [comment]
I wasn't sure if I should be changing the `cloudPlatform` type (`Option[WorkspaceCloudPlatform]`) to a non-optional if it's listed as `required`. How relevant is this to swagger? https://github.com/broadinstitute/rawls/blob/yp-wor-646-change/model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala#L736

**Review by marctalbott — APPROVED** [review]

File: core/src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -5156,6 +5156,9 @@ components:
         cloudPlatform:
           type: string
           description: The cloud platform of the workspace
+          enum:
```
Comment (cahrens) [inline core/src/main/resources/swagger/api-docs.yaml:5159]: I'm not sure whether this should be under "required" or not… I was going off the fact that we always now include this information in the response. @marctalbott, how do folks decide what to put under "required"?

Comment (cahrens) [reply core/src/main/resources/swagger/api-docs.yaml:5159]: Ah never mind, I saw @marctalbott's response in the other PR.

**Review by cahrens — APPROVED** [review]

**Pull request merged**